tests/unit/core/server/test_base.js

```diff
@@ -592,9 +592,9 @@ describe(__filename, () => {
       expect(auth_cookie).toContain(
         `${defaultConfig.get('cookieName')}=; Max-Age=0;`,
       );
-      expect(auth_cookie).toContain('HttpOnly');
-      expect(auth_cookie).toContain('Secure');
-      expect(auth_cookie).toContain('Lax');
+      expect(auth_cookie).toContain('Domain=.addons.mozilla.org;');
```

Comment:
I do not use the config value to make sure the config evaluates the template string correctly.